pkg/dashboard/templates/cost_settings.gohtml

```diff
@@ -0,0 +1,51 @@
+{{ define "cost_settings" }}
+<aside class="cost-settings-box" id="cost-settings-box">
+  <details open>
+    <summary class="--hideMarker">
+      <div class="cost-settings__title-container">
+        <p class="cost-settings-box__toggle-title">
+          Cost Settings
+          <i aria-hidden="true" class="fas fa-fw fa-angle-down --showWhenClosed"></i
+          ><i aria-hidden="true" class="fas fa-fw fa-angle-up --showWhenOpen"></i>
+        </p>
+        <div>
+         <button id="cost-settings__disable-cost-settings">Disable Cost Settings</button>
```

Comment:
This button living inside of the summary element isn't semantic HTML because it puts a button inside of a button. The "Disable Cost Settings" button needs to be outside of this summary element.

You can notice the negative effects of this both visually when you go to click the button and it looks like you're collapsing the form, as well as when you use a screenreader and it reads "Cost settings - disable cost settings" all at once.